src/main/java/teammates/ui/controller/StudentCommentsPageData.java

```diff
@@ -149,13 +150,14 @@ private void setCommentsForStudentsTables(String studentEmail, CourseRoster rost
         
         for (CommentAttributes comment : comments) {
             String recipientDetails = getRecipientNames(comment.recipients, courseId, studentEmail, roster);
+            String unsanitizedRecipientDetails = StringHelper.recoverFromSanitizedText(recipientDetails);
             InstructorAttributes instructor = roster.getInstructorForEmail(comment.giverEmail);
             String giverDetails = comment.giverEmail;
             if (instructor != null) {
                 giverDetails = instructor.displayedName + " " + instructor.name;
             }
-            
-            Comment commentRow = new Comment(comment, giverDetails, recipientDetails);
+            String unsanitizedGiverDetails = StringHelper.recoverFromSanitizedText(giverDetails);
+            Comment commentRow = new Comment(comment, unsanitizedGiverDetails, unsanitizedRecipientDetails);
```

Comment:
Where does this get sanitized now? It's very hard to look for files on the new github pr page.